src/components/InvoicesList/InvoicesList.test.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { render, cleanup, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { AppServicesProvider } from '../../services/pure-di';
+import { BrowserRouter } from 'react-router-dom';
+import InvoicesList from './InvoicesList';
+
+describe('InvoiceList component', () => {
+  afterEach(cleanup);
+
+  it('should show invocies', async () => {
+    // Arrange
+    const invoicesCollection = {
+      items: [
+        {
+          accountId: 'CD0000000073689',
+          product: 'Prod 1',
+          date: '09/25/2020',
+          currency: 'ARS',
+          amount: 1500,
+          filename: 'invoice_2020-10-05_10.pdf',
+        },
+        {
+          accountId: 'CD0000000073690',
+          product: 'Prod 2',
+          date: '09/26/2020',
+          currency: 'ARS',
+          amount: 1500,
+          filename: 'invoice_2020-10-05_10.pdf',
+        },
+      ],
+      totalItems: 2,
+    };
+
+    const dopplerBillingApiClientDouble = {
+      getInvoices: async () => {
+        return { success: true, value: invoicesCollection };
+      },
+    };
+
+    // Act
+    const { getByText } = render(
+      <AppServicesProvider
+        forcedServices={{
+          dopplerBillingApiClient: dopplerBillingApiClientDouble,
+        }}
+      >
+        <BrowserRouter>
+          <InvoicesList />
+        </BrowserRouter>
+      </AppServicesProvider>,
+    );
+    // Assert
+    await waitFor(() => expect(getByText('CD0000000073689')).toBeInTheDocument());
+  });
+
+  it('should show empty message', async () => {
+    // Arrange
+    const emptyCollection = { items: [], totalItems: 0 };
+    const dopplerBillingApiClientDouble = {
+      getInvoices: async () => {
+        return { success: true, value: emptyCollection };
+      },
+    };
+
+    // Act
+    const { getByText } = render(
+      <AppServicesProvider
+        forcedServices={{
+          dopplerBillingApiClient: dopplerBillingApiClientDouble,
+        }}
+      >
+        <BrowserRouter>
+          <InvoicesList />
+        </BrowserRouter>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    await waitFor(() => expect(getByText('No hay facturas para mostrar')).toBeInTheDocument());
```

Comment:
I have read that you are not going to add translations now. 
However, maybe it should be better to have the translations structure although we don't have the final content yet.
Beyond that, if you are not going to apply this changes here, keep in mind, that this text should be with the translation key to avoid problems if the text changes in a future.

Reply:
Luciana, I will include the translation in the current PR. Thanks for your feedback